topic/machine-learning/llm-langchain/requirements.txt

```diff
@@ -16,6 +16,6 @@ unstructured<0.17
 
 # Development.
 # cratedb-toolkit @ git+https://github.com/crate-workbench/cratedb-toolkit.git@main
-langchain @ git+https://github.com/crate-workbench/langchain.git@cratedb#subdirectory=libs/langchain
-langchain-community @ git+https://github.com/crate-workbench/langchain.git@cratedb#subdirectory=libs/community
+langchain @ git+https://github.com/crate-workbench/langchain.git@cratedb-v0.3.4#subdirectory=libs/langchain
+langchain-community @ git+https://github.com/crate-workbench/langchain.git@cratedb-v0.3.4#subdirectory=libs/community
```

Comment:
Just swapping the branch here for validation purposes. If CI succeeds on it, it will be converged into the canonical `cratedb` again.

Reply:
The canonical patch has been updated/refreshed, based on LangChain release 0.3.4.

- https://github.com/crate-workbench/langchain/pull/1